ports/bzip2/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "bzip2",
+  "version-string": "1.0.8",
+  "port-version": 1,
+  "description": "High-quality data compressor.",
+  "homepage": "http://www.bzip.org/"
```

Please use this homepage, because http://www.bzip.org is not up-to-date:
```suggestion
  "homepage": "https://sourceware.org/bzip2/"
```

---

Because bzip.org is the homepage and spourceware is a download page... bzip.org gives the history and description and sourceware has the downloads.

---

@StarGate-One Please have a look at:
**bzip.org changes hands**
https://lwn.net/Articles/762264/

---

I updated, but sourceware.org needs to file for an updated SPDX license expression as I had to use "bzip2-1.0.6" for vcpkg to accept it.